webpack/scenes/ContainerImages/ContainerImagesPage.scss

```
@@ -1,6 +1,34 @@
-.container-images-page {
-  // Remove gap below tabs
-  .pf-v5-c-tabs__list {
+#container-images-title-section {
+  .pf-v5-c-title {
+    padding-top: 0;
+    padding-bottom: 0;
     margin-bottom: 0;
+    margin-top: 0;
+    margin: 0;
```

Stylelint (17.14.0)

[error] Overridden property "margin-top" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)